src/pixiUtils.ts

```diff
@@ -5,39 +5,12 @@ const PX_PER_EM = 16;
 const PX_PER_PERCENT = 16 / 100;
 const PX_PER_PT = 1.3281472327365;
 
-export const measureFont = (context: CanvasRenderingContext2D): IFontMetrics =>
-  PIXI.TextMetrics.measureFont(context.font);
-
 export const INITIAL_FONT_PROPS: IFontMetrics = {
   ascent: 10,
   descent: 3,
   fontSize: 13,
 };
 
-// TODO: Memoize
-export const getFontPropertiesOfText = (
```

Comment:
Deleting this is causing tests to fail. Please fix or disable the tests if they're no longer relevant.